besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -2127,26 +2127,26 @@ public DataStorageConfiguration getDataStorageConfiguration() {
 
     if (SyncMode.FULL.equals(getDefaultSyncModeIfNotSet())
         && DataStorageFormat.BONSAI.equals(dataStorageConfiguration.getDataStorageFormat())
-        && dataStorageConfiguration.getBonsaiLimitTrieLogsEnabled()) {
+        && dataStorageConfiguration.getDiffbasedLimitTrieLogsEnabled()) {
```

Comment:
why is there a `Diffbased` prefix? It feels like this `DataStorageConfiguration` is doing more than one thing at the same time